app/templates/components/forms/order-form.hbs

```diff
@@ -0,0 +1,182 @@
+<form class="ui form" autocomplete="off" {{action 'submit' on='submit' }}>
+  <div class="ui segments">
+    <div class="ui aligned secondary segment">
+      <h3 class="weight-400">{{t 'Registration Information'}}</h3>
+    </div>
+    <div class="ui yellow inverted segment">
+      <div class="ui stackable grid">
+        <div class="two wide column" style="font-size:24px;padding:20px 10px 10px 20px">
+          08:48
+        </div>
+        <div class="fourteen wide column">
+          <p>
+            {{t 'Please complete registration within 10:00 minutes.'}}
+            <br>
+            {{t 'After 10:00 minutes, the reservation we&#39;re holding will be released to others.'}}
+          </p>
+        </div>
+      </div>
+    </div>
+    <h4 class="ui horizontal divider header">
+      <i class="ticket icon"></i>
+      {{t 'Ticket Buyer'}}
+    </h4>
+    <div class="ui equal width stackable center aligned page grid">
+      <div class="equal width row">
+        <div class="right floated right aligned column">
+          <label class="required" for="firstname">{{t 'First Name'}}</label>
+        </div>
+        <div class="column">
+          {{input type='text' id='firstname' placeholder=(t 'First Name')}}
+        </div>
+      </div>
+      <div class="equal width row">
+        <div class="right floated right aligned column">
+          <label class="required" for="lastname">{{t 'Last Name'}}</label>
+        </div>
+        <div class="column">
+          {{input type='text' id='lastname' placeholder=(t 'Last Name')}}
+        </div>
+      </div>
+      <div class="equal width row">
+        <div class="right floated right aligned column">
+          <label class="required" for="email">{{t 'Email'}}</label>
+        </div>
+        <div class="column">
+          {{input type='text' id='email' placeholder=(t 'Email')}}
+        </div>
+      </div>
+    </div>
+    <h4 class="ui horizontal divider header">
+      <i class="ticket icon"></i>
+      {{t 'Ticket Holder&#39;s Information'}}
+    </h4>
+    <div class="ui equal width stackable center aligned page grid">
+      <div class="equal width row">
+        <div class="column">
+        </div>
+        <div class="left floated left aligned column">
+          <i class="user icon"></i>
+          {{t 'Student/In, Ticket Holder1'}}
+        </div>
+      </div>
+      <div class="equal width row">
+        <div class="right floated right aligned column">
+          <label class="required" for="first-name">{{t 'First Name'}}</label>
+        </div>
+        <div class="column">
+          {{input type='text' id='first-name' placeholder=(t 'First Name')}}
```

Comment:
Don't use a placeholder for all the form fields. Doesn't make sense to have the same text in both label and placeholder